docs/detections/rules-coverage.asciidoc

```diff
@@ -1,9 +1,51 @@
 [[rules-coverage]]
 == MITRE ATT&CK® coverage
 
-:frontmatter-description: 
+:frontmatter-description: Review your current coverage of MITRE ATT&CK® tactics and techniques, based on installed rules.
 :frontmatter-tags-products: [security]
 :frontmatter-tags-content-type: [how-to]
-:frontmatter-tags-user-goals: [manage]
+:frontmatter-tags-user-goals: [manage, analyze, visualize]
 
-This is a placeholder for future documentation.
+The **MITRE ATT&CK® coverage** page shows which https://attack.mitre.org[MITRE ATT&CK®] adversary tactics and techniques are covered by your installed and enabled detection rules. This includes both Elastic prebuilt rules and custom rules.
+
+Mirroring the MITRE ATT&CK® framework, columns represent major tactics, and cells within each column represent a tactic's related techniques. Cells are darker when more rules are enabled for each technique, as indicated in the **Legend** at the top.
```

Comment:
I guess just one nit: a cell is darker when more rules matching the current filters fall under a given cell. This means:

- if only enabled rules are selected in the filter bar, "Cells are darker when more rules are enabled for each technique"
- if both enabled and disabled rules are selected (or none of them), "Cells are darker when more rules are installed for each technique"